packages/story-editor/src/utils/useIdleTaskQueue.ts

```diff
@@ -23,15 +23,20 @@ import { useRef, useCallback } from '@googleforcreators/react';
  */
 import { requestIdleCallback, cancelIdleCallback } from './idleCallback';
 
+interface Task {
+  taskId: string | null;
+  task: number | null | (() => Promise<void>);
```

Comment:
Because it changes from a promise to `idleCallbackId`, which is an int. That int is then passed to `clearQueuedTask` which expects an int. 

Really hard to follow, I know, took me a while to understand.